vep-mapping-pipeline/run_consequence_mapping.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 set -euxo pipefail
 
-# A wrapper script for running the consequence mapping pipeline concurrently and based on a VCF input.
+# A wrapper script for running the consequence mapping pipeline concurrently, based on ClinVar XML input.
 # See /README.md for detailed information.
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-bcftools query "$1" -f '%CHROM:%POS:%REF:%ALT\n' \
+# If you're running on VCF, substitute the first line with: bcftools query "$1" -f '%CHROM:%POS:%REF:%ALT\n'
+python3 "$DIR/extract_variants_for_vep.py" --clinvar-xml "$1" \
```

Comment:
Why not ${DIR} as it is in the code segment further down below?

Reply:
Good point, fixed: https://github.com/EBIvariation/eva-opentargets/pull/167/commits/7b455c2a2b53219920b7c691eabe2c3d3971ff2c